src/index.js

```diff
@@ -225,13 +225,12 @@ export default class HyperScreen extends React.Component {
    */
   render() {
     if (this.state.error) {
-      return (
-        <LoadError
-          error={this.state.error}
-          onPressReload={() => this.reload()}
-          onPressViewDetails={(uri) => this.props.openModal({url: uri})}
-        />
-      );
+      const errorScreen = this.props.errorScreen || LoadError;
+      return React.createElement(errorScreen, {
+        error: this.state.error,
+        onPressReload: () => this.reload(),
```

Comment:
This prevents creating a new function on every render, and would allow the custom error component to pass arguments to the original `reload` method (which accepts an optional URL)
```suggestion
        onPressReload: this.reload,
```